data/pathfinder/paizo/roleplaying_game/advanced_class_guide/acg_spells.lst

```diff
@@ -109,9 +109,9 @@ Slowing Mud					CLASSES:Druid,Shaman=4										SCHOOL:Conjuration	SUBSCHOOL:Cre
-Spellcrash					CLASSES:Cleric,Sorcerer,Wizard,Witch=6							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.193	DESC:This spell functions as lesser spellcrash, except that it affects 5th-level or lower prepared spell or spell slot.|PRERULE:1,DisplayFullSpell
-Spellcrash (Greater)			CLASSES:Cleric,Sorcerer,Wizard,Witch=8							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.194	DESC:This spell functions as lesser spellcrash, except that it affects a 7th-level or lower prepared spell or spell slot.|PRERULE:1,DisplayFullSpell
-Spellcrash (Lesser)			CLASSES:Cleric,Sorcerer,Wizard,Witch=4							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:instantaneous								SAVEINFO:Will negates													SPELLRES:yes					SOURCEPAGE:p.194	DESC:You create a discordant blast of energy that disrupts the target's available magic. If the target prepares spells, it must choose one of its prepared 3rd-level spells, which is immediately lost; if the target has no 3rd-level spells prepared, it must lose a 2ndlevel spell it has prepared (progressing down to 1st-level spell if it has no 2nd-level spells prepared). If the target is a spontaneous spellcaster, it loses one of its available 3rd-level spell slots; if the target has no available 3rd-level spell slots, it must lose a 2nd-level spell slot (progressing down to 1st-level spell if it has no 2ndlevel spell slots available). If the target has more than one spellcasting class, choose one at random to be affected. This spell has no effect on spell-like abilities. Any spell or spell slot lost because of this spell is treated as if the caster had failed a concentration check while trying to cast it-the spell or spell slot is wasted and has no effect, but is recovered normally the next time the character prepares spells or regains spell slots.|PRERULE:1,DisplayFullSpell
+Spellcrash					CLASSES:Cleric,Sorcerer,Wizard,Witch=6							SCHOOL:Abjuration														COMPS:V, S																					CASTTIME:1 standard action	RANGE:Medium					TARGETAREA:one creature																						DURATION:(CASTERLEVEL) rounds								SAVEINFO:Will negates (see text)													SPELLRES:yes					SOURCEPAGE:p.193	DESC:This spell functions as lesser spellcrash, except that it affects 5th-level or lower prepared spells or spell slots.|PRERULE:1,DisplayFullSpell
```

Comment:
There is an unexplained "space" that appears to be a tab or two for the spellcrash spells, can you verify this is not a space.